cmd/celestia/version.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildTime       string
+	lastCommit      string
+	semanticVersion string
+)
+
+func init() {
+	versionCmd.AddCommand()
```

Comment:
Seems like this code and init do nothing. Can you remove it?

Reply:
Done! Thanks for notice this.